test/integration/EnablePluginFromThis_TEST.cc

```diff
@@ -136,8 +134,6 @@ TEST(EnablePluginFromThis, LibraryManagement)
   }
 
   EXPECT_TRUE(weak.IsExpired());
-
-  CHECK_FOR_LIBRARY(libraryPath, false);
```

Comment:
I believe these checks are here to make sure the library is unloaded correctly. But @mxgrey's comment on the function suggests that it may not be reliable, so that could explain why you had to remove these. What do you think @mxgrey , should we remove the false checks, or adjust the `CHECK_FOR_LIBRARY` function somehow?

https://github.com/ignitionrobotics/ign-plugin/blob/07c20cbc61007bf0982583b3eecb7761ca5cf7c9/test/integration/utils.hh#L32-L42